fitbenchmarking/systests/test_regression.py

```diff
@@ -23,6 +23,7 @@ def setUpClass(cls):
 
         opts = Options()
         opts.minimizers = {k: [v[0]] for k, v in opts.minimizers.items()}
+        opts.software = sorted(opts.minimizers.keys())
```

Comment:
could you add comment to why you sort keys here please?

Reply:
done